bundles/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/model/KodiSubtitle.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.kodi.internal.model;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Class representing a Kodi subtitle stream (see https://kodi.wiki/view/JSON-RPC_API/v9#Player.Subtitle)
+ *
+ * @author Meng Yiqi - Initial contribution
+ */
+@NonNullByDefault
+public class KodiSubtitle {
+    private int index;
+    private @NonNullByDefault({}) String language;

```

Comment:
Do not use `@NonNullByDefault({})` for properties like this. We introduced the usage of `@NonNullByDefault({})` for variables we know that they are not null but this cannot be detected by the tooling. One common example has been references of components that are injected by the declarative service implementation on activation.

Rather set them to `""` if they should not be `null`.

```suggestion
    private String language = "";
```